example/README.md

```diff
@@ -25,15 +28,9 @@ This example aims to address the points above in a basic manner.
 
 ## Content
 * `deploy-operator.sh`: Deploy network operator and related resources.
-* `deploy-rdma-net.sh`: Deploy secondary network for RDMA traffic with static IPAM.
-* `deploy-rdma-net-ipam.sh`: Deploy secondary network for RDMA traffic with cluster IPAM
-([wherabouts](https://github.com/openshift/whereabouts-cni)).
 * `delete-operator.sh`: Delete network operator and related resources.
-* `delete-rdma-net.sh`: Delete secondary network for RDMA traffic with static IPAM.
-* `delete-rdma-net-ipam.sh`: Delete secondary network for RDMA traffic with cluster IPAM
-([wherabouts](https://github.com/openshift/whereabouts-cni)).
 * `deploy`: network-operator related deployment files
-* `networking`: kubernetes secondary network and IPAM related deployment files 
+* `networking`: kubernetes secondary network and IPAM related deployment files for hostdev
```

Comment:
how do i deploy macvlan network ? i see you deleted deploy-rdma-net and deploy-rdma-net-ipam ?

Reply:
perhaps use the macvlannetwork CR and have two files, one for macvlan with wherabouts and one without.

Reply:
macvlan can be deployed from the example/deply/crds example

Reply:
Added exmaple for both cases